MediaBrowser.MediaEncoding/Subtitles/SubtitleEditParser.cs

```diff
@@ -38,7 +38,7 @@ public SubtitleTrackInfo Parse(Stream stream, CancellationToken cancellationToke
             subRip.LoadSubtitle(subtitle, lines, "untitled");
             if (subRip.ErrorCount > 0)
             {
-                _logger.LogError("{ErrorCount} errors encountered while parsing subtitle.");
+                _logger.LogError("{ErrorCount} errors encountered while parsing subtitle.", subRip.ErrorCount);
```

Comment:
I believe logs should not end with full stops?

Reply:
Correct